contrib/neml.mk

```diff
@@ -11,11 +11,12 @@ neml_includes       := $(neml_DIR)/src
 neml_dep_includes   := -I$(neml_DIR)/rapidxml -I$(BLACKBEAR_DIR)/contrib/neml_extra_include
 
 $(APPLICATION_DIR)/lib/libblackbear-$(METHOD).la: $(neml_LIB)
+app_LIBS += $(neml_LIB)
```

Comment:
@permcody This PR is identical to the one that you proposed, except that I'm adding this line as well. It looks like it failed because the modules libraries are trying to link with neml, but they are trying to link before the neml library links, so this should register their dependency on neml so that won't happen. We'll see if it works...